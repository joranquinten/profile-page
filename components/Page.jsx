import React, { Component } from 'react';
import PropTypes from 'prop-types';

import Meta from './Meta';

import AppStyled from '../styled/App';

class Page extends Component {
  render() {
    const { children } = this.props;

    return (
      <>
        <Meta />
        <AppStyled>{children}</AppStyled>
      </>
    );
  }
}

Page.propTypes = {
  children: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.node]).isRequired,
};

export default Page;
