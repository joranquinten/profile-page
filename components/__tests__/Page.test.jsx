import { shallow } from 'enzyme';

import Page from '../Page';

// jest.mock('../../data/nl.js');

describe('<Page />', () => {
  it('should render without crashing', () => {
    expect(shallow(<Page>Child</Page>)).toMatchSnapshot();
  });
});
