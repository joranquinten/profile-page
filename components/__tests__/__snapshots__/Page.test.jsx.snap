// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> should render without crashing 1`] = `
<Fragment>
  <Meta />
  <App>
    Child
  </App>
</Fragment>
`;
